src/main/java/com/indeed/jiraactions/JiraActionsIndexBuilderCommandLine.java

```diff
@@ -85,7 +87,11 @@ private void initialize(String[] args) {
             final String iuploadUsername = config.getString("iupload.username");
             final String iuploadPassword = config.getString("iupload.password");
             final String indexName = config.getString("indexname");
-
+            final boolean buildJiraIssues = config.getBoolean("buildjiraissues");
+            final int jiraIssuesRange = config.getInt("jiraissuesrange");
+            final String[] deliveryLeadTimeStatuses = {"Accepted", "In Progress", "Reopened", "In Development", "In Dev Blocked", "Pending Review", "Pending Code Review", "Pending Dependencies", "Pending Merge", "Pending QA Release", "Conflict", "Pending Verification", "In QA", "Final Verification", "QA Ready", "Pending Closure", "Pending Prod Release", "In Production"};
```

Comment:
This is better, but can we read them from a config file instead of being specified on the command line?